doc/design/show_train_stmt.md

```diff
@@ -0,0 +1,90 @@
+# Show Train Statement in SQLFlow
+
+## Background
+Like using other SQL engines, people may want to access the table metadata (say, the `CREATE TABLE ` statement) using a SQL query. Usually they will type a statement like:
+```
+SHOW CREATE TABLE `test`;
+```
+which will produce:
+```
+mysql> SHOW CREATE TABLE `test`;
++-------+------------------------------------------------------------------+
+| Table | Create Table                                                                                                                                                                                                                                                          |
++-------+------------------------------------------------------------------+
+| test  | CREATE TABLE `test` (
+  `sepal_length` float DEFAULT NULL,
+  `sepal_width` float DEFAULT NULL,
+  `petal_length` float DEFAULT NULL,
+  `petal_width` float DEFAULT NULL,
+  `class` int DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci |
++-------+-----------------------------------------------------------------+
+```
+
+It worth for SQLFlow to do the same thing. On SQLFlow platform, user may create a lot of models with a minor set of parameters different with each other. After a while, the user may hardly tell what the differences are if he hadn't record the information else where (like in model name). We would like to provide a formal way to solve this problem. Here we introduce the `SHOW TRAIN` feature.
+
+## Grammar
```

Comment:
I prefer using `Syntax` instead of `Grammer`, because

> The syntax of a programming language is the set of all syntactically valid programs.
A grammar is one way to specify a formal language.

Grammar is context-free.

Reply:
👍